patches/ungoogled-chromium/macos/fix-disabling-safebrowsing.patch

```diff
@@ -77,30 +79,56 @@
    defines = []
    libs = []
  
-@@ -400,7 +394,6 @@ jumbo_split_static_library("ui") {
+@@ -377,7 +371,6 @@ jumbo_split_static_library("ui") {
+   # browser, then we can clean up these dependencies.
+   public_deps = [
+     "//components/dom_distiller/core",
+-    "//components/safe_browsing:buildflags",
+     "//components/sync",
+     "//components/sync_user_events",
+     "//components/translate/content/browser",
+@@ -403,7 +396,6 @@ jumbo_split_static_library("ui") {
      "//chrome/browser/media:mojo_bindings",
      "//chrome/browser/notifications/scheduler/public",
      "//chrome/browser/profiling_host",
 -    "//chrome/browser/safe_browsing",
      "//chrome/browser/ssl:proto",
      "//chrome/browser/ui/webui/bluetooth_internals",
      "//chrome/browser/ui/webui/downloads:mojo_bindings",
-@@ -479,14 +472,6 @@ jumbo_split_static_library("ui") {
+@@ -482,16 +474,6 @@ jumbo_split_static_library("ui") {
      "//components/rappor",
      "//components/renderer_context_menu",
      "//components/resources",
+-    "//components/safe_browsing:csd_proto",
 -    "//components/safe_browsing:features",
 -    "//components/safe_browsing/common",
 -    "//components/safe_browsing/common:safe_browsing_prefs",
 -    "//components/safe_browsing/db:database_manager",
 -    "//components/safe_browsing/db:util",
 -    "//components/safe_browsing/password_protection",
+-    "//components/safe_browsing/password_protection:password_protection_metrics_util",
 -    "//components/safe_browsing/triggers:ad_popup_trigger",
 -    "//components/safe_browsing/triggers:ad_redirect_trigger",
      "//components/safe_browsing/web_ui",
      "//components/search",
      "//components/search_engines",
-@@ -3823,15 +3808,6 @@ jumbo_split_static_library("ui") {
+@@ -1359,7 +1341,6 @@ jumbo_split_static_library("ui") {
+       "//chrome/browser/profile_resetter:profile_reset_report_proto",
+       "//chrome/browser/resource_coordinator:tab_metrics_event_proto",
+       "//chrome/browser/resource_coordinator/tab_ranker",
+-      "//chrome/browser/safe_browsing:advanced_protection",
```

Comment:
Since leaving this in doesn't seem an option, every reference to `AdvancedProtectionStatusManager` will have to be removed (6-7).

Reply:
That sounds reasonable to me. If it belongs in the `safe_browsing` namespace, it probably has to go.